### DISCUSSION TITLE
Running two video players at the same time

### DISCUSSION THREAD

**Discussion by orenzav** [PaulleDemon/tkVideoPlayer#50] [Q&A]
Is there a recommended way to run two instances of the player at once? I have two working, one in a main window and the other in a top level window but they run very slow. Any help would be appreciated

**Comment by PaulleDemon** [comment]
I can't tell how slow its running without an reproducible example or video, but the below code did fine on small video clip. 
```py
import tkinter as tk
from tkVideoPlayer import TkinterVideo


def top():

    toplevel = tk.Toplevel(root)
    

    videoplayer2 = TkinterVideo(master=toplevel, scaled=True)
    videoplayer2.load(r"test.mp4")
    videoplayer2.pack(expand=True, fill="both")
    videoplayer2.play() # play the video


root = tk.Tk()

videoplayer = TkinterVideo(master=root, scaled=True)
videoplayer.load(r"test.mp4")
videoplayer.pack(expand=True, fill="both")

videoplayer.play() # play the video

top()

root.mainloop()
```

**Reply by orenzav** [reply]
Thanks, the two videos I'm playing are 640x360. Both appear fine when played at very small size and one runs fine at correct size when played by itself, but when I play scaled up to the above size I'm seeing stuttering and frame drops, looking at a slow down of 2-3 seconds over a 25sec clip. Would this be expected? Running on a pi4
